### PR TITLE
Fix a few phpcs errors

### PR DESCRIPTION
**Pull request by flow-control**

### CODE REVIEW

File: source/Core/Session.php [hunk 1]
```diff
@@ -199,19 +199,15 @@ public function getName()
     }
 
     /**
-     * Starts shop session, generates unique session ID, extracts user IP.
+     * retrieves the session id from the request if any
+     *
+     * @return string|null
      */
-    public function start()
+    protected function _getSidFromRequest()
```
Comment (Sieg) [inline source/Core/Session.php]: Function name doesn't fit new coding style, it should be without underscore.

File: source/Application/Controller/OrderController.php [hunk 2]
```diff
@@ -519,7 +519,7 @@ protected function _getNextStep($iSuccess)
             case ($iSuccess === \OxidEsales\Eshop\Application\Model\Order::ORDER_STATE_MAILINGERROR):
                 $sNextStep = 'thankyou?mailerror=1';
                 break;
-            case ($iSuccess === \OxidEsales\Eshop\Application\Model\Order::ORDER_STATE_INVALIDDElADDRESSCHANGED):
+            case ($iSuccess === \OxidEsales\Eshop\Application\Model\Order::ORDER_STATE_INVALIDDELADDRESSCHANGED):
```
Comment (Sieg) [inline source/Application/Controller/OrderController.php]: We cannot change it so easily, if we remove old constant, its BC break.

File: source/Application/Model/FileChecker.php [hunk 3]
```diff
@@ -50,7 +50,7 @@ class FileChecker
      *
      * @var string
      */
-    public $_sWebServiceUrl = 'http://oxchkversion.oxid-esales.com/webService.php';
+    protected $_sWebServiceUrl = 'http://oxchkversion.oxid-esales.com/webService.php';
```
Comment (Sieg) [inline source/Application/Model/FileChecker.php]: BC break

File: source/Core/Database/Adapter/Doctrine/Database.php [hunk 4]
```diff
@@ -926,13 +927,10 @@ protected function convertException(\Exception $exception)
                 /** @var $pdoException PDOException */
                 $pdoException = $exception->getPrevious();
 
-                if ($pdoException->errorInfo[1]) {
+                if ($pdoException instanceof PDOException) {
```
Comment (Sieg) [inline source/Core/Database/Adapter/Doctrine/Database.php]: But this is the switch case for DBALException, have i missed something?

**Comment by flow-control** [comment]
shit, ich hab was falsch gemacht ;-)

**Comment by flow-control** [comment]
ok, i fixed it :-)

**Comment by Sieg** [comment]
Thank you for your analysis and work. I have rebased and squashed a little your commits. Merged the changes but skipped the 4df8ad7 commit.

**Pull request closed without merging**